Resources/config/services.xml

```diff
@@ -7,6 +7,7 @@
         <!-- commands -->
         <service id="doctrine.fixtures_load_command" class="Doctrine\Bundle\FixturesBundle\Command\LoadDataFixturesDoctrineCommand">
             <argument type="service" id="doctrine.fixtures.loader" />
+            <argument type="service" id="doctrine" on-invalid="null" />
```

Comment:
```suggestion
            <argument type="service" id="doctrine" />
```

Since this bundle has a hard dependency on doctrine-bundle, not having a `doctrine` service should not silently fail but rather fail hard. Was there a particular reason why you added it?

Reply:
Yes, because integration tests don't include it, but I think you are right and tests should be fixed.

Reply:
done.